server/app.py

```
@@ -0,0 +1,293 @@
+    """
+    tree = load_tree(request.conversation_id)
+    
+    async def generate():
+        try:
+            for chunk in tree.chat_stream(request.message, provider=request.provider, model=request.model):
+                yield chunk
+            
+            # Save after completion
+            path = get_file_path(request.conversation_id)
+            tree.save_to_file(path)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)